dashboard-frontend/src/types/study.ts

```diff
@@ -0,0 +1,33 @@
+export interface Study {
+  id: string;
+  name: string;
+  slug: string;
+  createdAt: string;
+  progress: StudyProgress;
+  strata: Strata[];
+}
+
+interface StudyProgress {
+  currentParticipants: number;
+  expectedParticipants: number;
+  currentAverageDeviation: number;
+  expectedAverageDeviation: number;
+  date: string;
+  averageDeviationOverTime: DataPoint[];
+}
+
+interface Strata {
```

Comment:
I would call this `StratumProgress`, in line with `StudyProgress`.

Reply:
Yes, I like it 👍